core/src/main/scala/com/avast/clients/storage/compression/ZstdDecompressOutputStream.scala

```diff
@@ -0,0 +1,52 @@
+package com.avast.clients.storage.compression
+
+import com.github.luben.zstd.{ZstdDecompressCtx, ZstdInputStream}
+
+import java.io.OutputStream
+import java.nio.ByteBuffer
+import java.nio.channels.Channels
+
+class ZstdDecompressOutputStream(outputStream: OutputStream) extends OutputStream {
+  private val decompressCtx = new ZstdDecompressCtx()
+  private val outputChannel = Channels.newChannel(outputStream)
+  private val closed = false
```

Comment:
I bet you don't set this value anywhere anymore... 🙃

Reply:
Fixed.